emulate_user.less

```diff
@@ -0,0 +1,62 @@
+.htk-toolbar {
+    bottom: 0;
+    line-height: 1.5em;
+    position: fixed;
+    overflow-y: auto;
+    right: 0;
+    top: 0;
+    width: 200px;
+    z-index: 100000000;
+
+    opacity: 0.9;
+    background-color: #111;
+    .htk-toolbar-hide {
+        border-bottom: 1px solid #222;
```

Comment:
all of the hardcoded color codes here should be replaced with variables later

Reply:
Roger that